aspnetcore/blazor/forms-validation.md

```diff
@@ -236,16 +322,10 @@ private class MyCustomValidator : ValidationAttribute
 }
 ```
 
-::: moniker range=">= aspnetcore-3.1"
-
 ### Blazor data annotations validation package
 
 The [Microsoft.AspNetCore.Blazor.DataAnnotations.Validation](https://www.nuget.org/packages/Microsoft.AspNetCore.Blazor.DataAnnotations.Validation) is a package that fills validation experience gaps using the `DataAnnotationsValidator` component. The package is currently *experimental*.
 
-### [CompareProperty] attribute
```

Comment:
While the using the `CompareAttribute` isn't as broken in 3.1 as it was in 3.0 it still has issues because it doesn't associated the validation result with a specific member. This can result in inconsistent behavior between field-level validation and when the entire model is validated on a submit. The `ComparePropertyAttribute` addresses these issues and should still be recommended.